integration_tests/grpc_rest/pki/helpers.go

```diff
@@ -1292,4 +1391,210 @@ func Demo(suite *utils.TestSuite) {
+	require.Equal(suite.T, "label", revDistPointsBySubj.Points[0].Label)
+	require.Equal(suite.T, testconstants.SubjectKeyIDWithoutColons, revDistPointsBySubj.IssuerSubjectKeyID)
+
+	// Add revocation distribution point for PAI by Vendor
+	//	venName65522 := utils.RandString()
+	//	venAcc65522 := test_dclauth.CreateVendorAccount(
+	//		suite,
+	//		venName65522,
+	//		dclauthtypes.AccountRoles{dclauthtypes.Vendor},
+	//		65522,
+	//		jackName,
+	//		jackAccount,
+	//		aliceName,
+	//		aliceAccount,
+	//		testconstants.Info,
+	//	)
+	//	require.NotNil(suite.T, venAcc65522)
+	//
+	//	// Jack (Trustee) propose Root certificate
+	//	msgProposeAddX509RootCert = pkitypes.MsgProposeAddX509RootCert{
+	//		Cert:   testconstants.PAACertNoVid,
+	//		Signer: jackAccount.Address,
+	//		Vid:    venAcc65522.VendorID,
+	//	}
+	//	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&msgProposeAddX509RootCert}, jackName, jackAccount)
+	//	require.NoError(suite.T, err)
+	//	proposedCertificate, err = GetProposedX509RootCert(suite, testconstants.PAACertNoVidSubject, testconstants.PAACertNoVidSubjectKeyID)
+	//
+	//	// Alice (Trustee) approve Root certificate
+	//	secondMsgApproveAddX509RootCert = pkitypes.MsgApproveAddX509RootCert{
+	//		Subject:      proposedCertificate.Subject,
+	//		SubjectKeyId: proposedCertificate.SubjectKeyId,
+	//		Signer:       aliceAccount.Address,
+	//	}
+	//	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&secondMsgApproveAddX509RootCert}, aliceName, aliceAccount)
+	//	require.NoError(suite.T, err)
+	//
+	//	// Add PAI certificate
+	//	msgAddX509Cert = pkitypes.MsgAddX509Cert{
+	//		Cert:   testconstants.PAICertWithPidVid,
+	//		Signer: venAcc65522.Address,
+	//	}
+	//	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&msgAddX509Cert}, venName65522, venAcc65522)
+	//	require.NoError(suite.T, err)
+	//
+	//	msgAddPkiRevDistPoints = pkitypes.MsgAddPkiRevocationDistributionPoint{
+	//		Signer:               venAcc65522.Address,
+	//		Vid:                  venAcc65522.VendorID,
+	//		IssuerSubjectKeyID:   testconstants.SubjectKeyIDWithoutColons,
+	//		IsPAA:                false,
+	//		Pid:                  8,
+	//		CrlSignerCertificate: testconstants.PAICertWithVid,
+	//		Label:                "label_PAI",
+	//		DataURL:              testconstants.DataURL,
+	//		RevocationType:       1,
+	//	}
+	//	_, err = suite.BuildAndBroadcastTx([]sdk.Msg{&msgAddPkiRevDistPoints}, venName65522, venAcc65522)
+	//	require.NoError(suite.T, err)
+	//
+	//	revDistPoints, _ = GetAllPkiRevocationDistributionPoints(suite)
+	//	require.Equal(suite.T, 2, len(revDistPoints))
+	//
+	//	revDistPoint, _ = GetPkiRevocationDistributionPoint(suite, venAcc65522.VendorID, testconstants.SubjectKeyIDWithoutColons, "label")
+	//	require.Equal(suite.T, venAcc65522.VendorID, revDistPoint.Vid)
+	//	require.Equal(suite.T, "label_PAI", revDistPoint.Label)
+	//	require.Equal(suite.T, testconstants.SubjectKeyIDWithoutColons, revDistPoint.IssuerSubjectKeyID)
```

Comment:
Let's also call `GetPkiRevocationDistributionPointsBySubject` here. BTW it will return a list of two values, as both PAA and PAI revocation points created for the same `testconstants.SubjectKeyIDWithoutColons`